CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.3.0] 2020-09-01
```

Comment:
```suggestion
## [1.3.0] 2020-09-01 (UNPUBLISHED)
```

Reply:
Also all the rest of the changes please e.g., context related changes that were made in this release.

Reply:
oh ok, I must have misunderstood, I thought you were just referencing the ChannelList stuff, not all the Channel context stuff, but can add no problem!